angular/src/assets/appconfig.production.json

```diff
@@ -1,5 +1,5 @@
 {
-  "remoteServiceBaseUrl": "https://localhost:44311",
+  "remoteServiceBaseUrl": "http://localhost:9902",
```

Comment:
You exposed 44311 only and using 9902 here. It doesn't match here.

Reply:
The 44311 is only inside the container, which is mapped to 9902 on this file: 

aspnet-core/docker/ng/docker-compose-updated.yml

Reply:
After the update to default port 80. please see the map in this file: https://github.com/EduardoSantanaSeverino/module-zero-core-template/blob/c96ad561f39ee4f0c250106c0688b0d56f3d345c/aspnet-core/docker/ng/docker-compose.yml#L13

Angular is calling the API from an outside web browser, therefore calling the container from the outside port which is 9902.